docs/upgrading.rst

```diff
@@ -44,13 +44,13 @@ need to change only the first module of your import statements, not the submodul
     from cassandra.auth import PlainTextAuthProvider
     from cassandra.policies import WhiteListRoundRobinPolicy
 
-Also note that the cassandra.hosts module doesn't exist in cassandra-driver. This
+Also note that the cassandra.hosts module doesn't exist in scylla-driver. This
 module is named cassandra.pool.
 
 dse-graph
 ^^^^^^^^^
 
-dse-graph features are now built-in in cassandra-driver. The only change you need
+dse-graph features are now built-in in scylla-driver. The only change you need
```

Comment:
"... are now built into scylla-driver"

Reply:
I think I have enough of my own grammar mistakes to fix :), I'll change it

future merging me, might regret some those changes :face_with_head_bandage: